app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+  before_action :set_post
+  before_action :set_comment, only: [:destroy]
+
+  def create
+    @comment = @post.comments.new(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to @post, notice: 'You commented this post!'
+    elsif @comment.errors[:content].include?("can't be blank")
+      redirect_to @post, alert: 'You cannot post an empty comment, write something!'
```

Comment:
This seems more like a view than a controller concern, I'd add this to the view removing this `elsif` from the controller.